python/doc/examples/meta_modeling/general_purpose_metamodels/plot_expert_mixture.py

```diff
@@ -3,35 +3,33 @@
 ==================
 """
 # %%
-# In this example we are going to approximate a piece wise continuous function using an expert mixture of metamodels.
+# In this example we are going to approximate a piece-wise continuous function using an expert mixture of metamodels.
```

Comment:
This a single word (see [Wikipedia](https://en.wikipedia.org/wiki/Piecewise_function)):
```suggestion
# In this example we are going to approximate a piecewise continuous function using an expert mixture of metamodels.
```